.github/workflows/ci.yml

```diff
@@ -11,7 +11,15 @@ jobs:
     steps:
       - name: Check out repository code 
         uses: actions/checkout@v3
-
+      
+      - name: Setup env variables for streaming
```

Comment:
Netlify won't run this workflow, so you need to put these in an shell script and run it in 'prebuild' script of package.json